app/src/androidTest/java/com/willowtree/vocable/tests/BaseTest.kt

```diff
@@ -77,6 +79,8 @@ open class BaseTest {
         // Catch the exception if the popup doesn't appear so we don't fail the tests
         try {
             device.wait(Until.findObject(By.text("CONTINUE")), 10000).click()
+            Thread.sleep(2000)
+            device.pressBack()
```

Comment:
Do we need to press continue, and then back? I think we can just press back and the dialog will dismiss.

Reply:
No we do not. Pressing click just adds complexity. Removing the 'click' action also gets rid of the need to wait in the next step. Thanks